circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/relations/standard_relation_consistency.test.cpp

```diff
@@ -0,0 +1,148 @@
+/**
+ * @file standard_relation_consistency.test.cpp
+ * @brief Show that relation arithmetic has a simple form.
+ * @details The purpose of this test suite is to show that the identity arithmetic implemented in the Relations is
+ * equivalent to a simpler unoptimized version implemented in the tests themselves. This is useful 1) as documentation
+ * since the simple implementations here should make the underlying arithmetic easier to see, and 2) as a check that
+ * optimizations introduced into the Relations have not changed the result.
+ *
+ * For this purpose, we simply feed (the same) random inputs into each of the two implementations and confirm that
+ * the outputs match. This does not confirm the correctness of the identity arithmetic (the identities will not be
+ * satisfied in general by random inputs) only that the two implementations are equivalent.
+ *
```

Comment:
I've significantly reworked these relation consistency tests to simplify them, as they now don't need to test separate code paths (each relation implementation now contains a single function called `accumulate`).